pkg/errtypes/errtypes.go

```diff
@@ -145,6 +145,16 @@ func (e InsufficientStorage) Error() string { return "error: insufficient storag
 // IsInsufficientStorage implements the IsInsufficientStorage interface.
 func (e InsufficientStorage) IsInsufficientStorage() {}
 
+// StatusCode returns StatusInssufficientStorage, this implementation is neede to allow TUS to cast the correct http errors.
+func (e InsufficientStorage) StatusCode() int {
+	return StatusInssufficientStorage
```

Comment:
```suggestion
	return StatusInsufficientStorage
```

Is the name already spelt like that?